### PR TITLE
Bump clipboard to remove libc 0.1.z dependency

### PR DESCRIPTION
**Pull request by nox**
[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9473)

### CODE REVIEW

**Comment by Ms2ger** [comment]
Remove it from the whitelist?

**Comment by frewsxcv** [comment]
Remove `cocoa` while you're at it too

https://github.com/servo/servo/blob/46b3eb653579a40632f91497a3d48f1d7fbd40cc/python/tidy.py#L171

**Comment by nox** [comment]
Didn't know about this thing.

File: python/tidy.py [hunk 1]
```diff
@@ -178,9 +175,6 @@ def find_reverse_dependencies(dependency, version, content):
         packages.setdefault(package["name"], []).append(package["version"])
 
     for (name, versions) in packages.iteritems():
-        if name in exceptions or len(versions) <= 1:
```
Comment (KiChjang) [inline python/tidy.py]: Do we want to keep the `len(versions) <= 1` condition?

Comment (nox) [inline python/tidy.py]: Oh yes, probably.

**Comment by nox** [comment]
This is going to fail.

```
± ./mach test-tidy
./ports/cef/Cargo.lock:1: duplicate versions for package "cocoa"
	found dependency on version 0.1.5
	but highest version is 0.2.2
	try upgrading with ./mach cargo-update -p cocoa:0.1.5
	The following packages depend on version 0.1.5:
		embedding
```

**Pull request merged**